waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -145,12 +147,12 @@ npmDepsForWasp spec =
         AS.Dependency.fromList
           [ ("cookie-parser", "~1.4.6"),
             ("cors", "^2.8.5"),
-            ("express", "~4.18.1"),
+            ("express", expressVersionStr),
             ("morgan", "~1.10.0"),
-            ("dotenv", "16.0.2"),
+            ("dotenv", "^16.0.2"),
```

Comment:
Allows feat version bumps, why not, we do it for other deps.

Reply:
Hm yeah I am always wondering about this. On one hand, this means that at any point some of these deps can introduce a bug (they happen even in patches, so certaily also in minor updates). On the other hand, by fixing the version we haven't really solved that problem anyway, because of the deps of deps -> real solution to avoid unexpected updates messing stuff up is only by doing dep freezing/locking.
Ok, makes sense to keep it all same now (`^`) and then we will figure out in the future what is the best solution.